multipatch_analysis/metadata_submission.py

```diff
@@ -377,7 +379,9 @@ def _import_old_metadata(self, site_info, warnings, errors):
                         labels[color] = pos
                 else:
                     warnings.append("Pipette %d: old metadata has unrecognized label: %s." % (pid, label))
-            
+            # if target layer is not set above for human or mouse L2/3 assume layer 5
+            if pip['target_layer'] == '':
```

Comment:
`and self.spec_info['organism'] == 'mouse'`

Reply:
since we're automatically setting a lot of the layer information, should we call attention in the Warnings if it's NOT set so we know to go back?